frontend/src/app/components/nav/nav.component.css

```diff
@@ -1,140 +1,129 @@
 ::ng-deep :root {
-    font-size: 16px;
-    font-family: 'Open Sans';
-    --text-primary: black;
-    --text-secondary: darkgray;
-    --bg-primary: #C69AD6;
-    --bg-secondary: #8f669f;
-    --transition-speed: 600ms;
+  font-size: 16px;
+  font-family: "Open Sans";
+  --text-primary: black;
+  --text-secondary: darkgray;
+  --bg-primary: #c69ad6;
+  --bg-secondary: #8f669f;
+  --transition-speed: 600ms;
 }
 
 body {
-    color: black;
-    background-color: white;
-    margin: 0;
-    padding: 0;
+  color: black;
+  background-color: white;
+  margin: 0;
+  padding: 0;
 }
 
 body::-webkit-scrollbar {
-    width: 0.25rem;
+  width: 0.25rem;
 }
 
 body::-webkit-scrollbar-track {
-    background: #1e1e24;
+  background: #1e1e24;
 }
 
 body::-webkit-scrollbar-thumb {
-    background: #6649b8;
+  background: #6649b8;
 }
-
+/* 
 main {
-    margin-left: 5rem;
-    padding: 1rem;
-}
+  margin-left: 5rem;
+  padding: 1rem;
+} */
 
 .navbar {
   z-index: 100;
-    position: fixed;
-    background-color: var(--bg-primary);
-    width: max-content;
-    height: 100vh;
-    transition: width var(--transition-speed) ease;
-    left: 0px;
-    z-index: 20;
+  position: fixed;
+  background-color: var(--bg-primary);
+  width: 14vw;
```

Comment:
Make sure we want to use "vw" as the measurement type. px or em might be more appropriate for navbar. Not that vw is neccesarrily wrong